setup.cfg

```
@@ -1,4 +1,66 @@
-# This file should be used as the pep8/pycodestyle and autopep8 config
+###############################################################################
+# Setuptools
+###############################################################################
+
+[metadata]
+name = WebAppDIRAC
+description = WebAppDIRAC is a portal for the DIRAC software.
+long_description = file: README.rst
+url = https://github.com/DIRACGrid/WebAppDIRAC/
+license = GPL-3.0-only
+license_files = LICENSE
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Scientific/Engineering
+    Topic :: System :: Distributed Computing
+
+[options]
+python_requires = >=3.8
+package_dir=
+    =src
+packages = find:
+
+install_requires =
+    dirac
+    diraccfg
+    autopep8 ==1.3.3
+    coverage
+    requests >=2.9.1
+    six
+    mock >=1.0.1
+    pylint >=1.6.5
+    pytest >=3.6
+    pytest-cov >=2.2.0
+    pytest-mock
```

Many of these are test dependencies and `tornado` should be added as it's used directly. I don't think there is anything else but please check:
```suggestion
    requests >=2.9.1
    tornado
    six
```

---

Correct me, IIUC then WebAppDIRAC should be installed with the dependence of the `dirac` package and it has the dependence of `tornado`, do it still needed to specify both there and here?

---

You should depend on whatever WebAppDIRAC uses directly. The way to think about it is "if DIRAC suddenly stopped using tornado would WebAppDIRAC still need it?" and in this case the answer is yes.

---

Thanks, now I understand what you mean